kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -1019,37 +1030,25 @@ define([
                     ui.disableButton(_button);
                 });
             }
-
-            // Element state
-            if (state.ui.elements) {
-                state.ui.elements.show.forEach((element) => {
-                    ui.showElement(element);
-                });
-                state.ui.elements.hide.forEach((element) => {
-                    ui.hideElement(element);
-                });
-            }
         }
 
-        function doOnQueued(message) {
-            const noLogYet = {
-                lineNumber: '',
-                text: 'Job is queued, logs will be available when the job is running.',
-            };
-            const line = buildLine(noLogYet);
-            getLogPanel().appendChild(line);
+        function doOnQueued() {
+            ui.setContent(
+                'log-panel',
+                p(['Job is queued; logs will be available when the job is running.'])
+            );
```

Comment:
just add the message directly, don't faff around with pretending it is a log line